ic-agent/Cargo.toml

```diff
@@ -22,16 +22,16 @@ hex = "0.4.0"
 ic-types = { path = "../ic-types", version = "0.1", features = [ "serde" ] }
 leb128 = "0.2.4"
 mime = "0.3.16"
-num-bigint = "0.2.6"
-openssl = "0.10.24"
+num-bigint = "0.3.1"
```

Comment:
Still needed?

Reply:
Yes, it's still needed, because the `oid!` macro uses BigUInt. (I tried to remove it)